Q7/q7.csv

```diff
@@ -0,0 +1 @@
+5525
```

Comment:
This will be the valid input and output.   
I/P
10, 20
30, 60
50, 20
70, 20

O/P
10, 20, 200 
30, 60, 1800
50, 20, 1000
70, 20, 1400